src/axom/quest/DistributedClosestPoint.hpp

```diff
@@ -141,13 +146,17 @@ template <int NDIMS>
 void get_bounding_box_from_conduit_node(primal::BoundingBox<double, NDIMS>& bb,
                                         const conduit::Node& node)
 {
-  int dim = node.fetch_existing("dim").as_int();
-  SLIC_ASSERT(dim == NDIMS);
-  const double* lo = node.fetch_existing("lo").as_double_ptr();
-  const double* hi = node.fetch_existing("hi").as_double_ptr();
-  bb =
```

Comment:
Fun fact -- loading it this way was actually setting the "uninitialized" bounding box to have infinite extent ("lo" was the smallest double; "hi" was the largest double), instead of invalid bounds!  My bugfix resolves this.

Reply:
Thanks @kennyweiss